app/src/components/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import { usePathname, useRouter } from "src/i18n/navigation";
+
+import { useLocale } from "next-intl";
+import { CSSProperties, useTransition } from "react";
+import { LanguageDefinition, LanguageSelector } from "@trussworks/react-uswds";
+
+// Currently, the `react-uswds` component erroneously sets 'usa-language-container' class
+// on both the container and the button, which causes incorrect positioning relative to nav items
```

Comment:
Could we open a PR to fix that for them? They're usually pretty open to contributions.

Reply:
https://github.com/trussworks/react-uswds/pull/2893